tsl/test/sql/include/cagg_on_cagg_validations.sql

```diff
@@ -47,6 +47,10 @@ WITH NO DATA;
 \set ON_ERROR_STOP 1
 \set VERBOSITY terse
 
+-- Check for incorrect CAGGs
+\if :INTERVAL_TEST
+  SELECT COUNT(*) from :CAGG_NAME_2TH_LEVEL;
```

Comment:
This test not FAIL without the fix... the idea of adding a regression test is to exercise the failing code path.